apps/api/.gitignore

```diff
@@ -54,3 +54,9 @@ pids
 
 # Diagnostic reports (https://nodejs.org/api/report.html)
 report.[0-9]*.[0-9]*.[0-9]*.[0-9]*.json
+
+# Schema file
+.schema.gpl
```

Comment:
This is incorrect. Also verify your PRs before giving for review. We can still see the file and folder.

Reply:
Updated path and removed the files